### PR TITLE
Support higher serial baud rates

### PR DESCRIPTION
**Pull request by josephn123**
Add some further high baud rate options, and some logging.

### CODE REVIEW

File: src/util/socket-utils.c [hunk 1]
```diff
@@ -881,9 +906,15 @@ open_super_socket(const char* socket_name)
 		for (; NULL != options; options = strchr(options+1, ',')) {
 			if (strcasehasprefix(options, ",b") && isdigit(options[2])) {
 				// Change Baud rate
-				int baud = baud_rate_to_termios_constant((int)strtol(options+2,NULL,10));
+				const int baud = (int)strtol(options+2,NULL,10);
+				const int baud_opt = baud_rate_to_termios_constant(baud);
+				if(baud_opt <= 0) {
+					syslog(LOG_ERR, "Unsupported baud rate %d", baud);
```
Comment (josephn123) [inline src/util/socket-utils.c]: Thoughts on closing the file descriptor here and returning -1 instead of continuing?

**Comment by josephn123** [comment]
@abtink any thoughts on this PR?

**Comment by abtink** [comment]
@josephn123 , I am sorry, somehow totally missed this PR. LGTM.

File: src/util/socket-utils.c (hunk 1 above)
Comment (abtink) [reply src/util/socket-utils.c]: I agree. Returning -1 sound good here.

Comment (josephn123) [reply src/util/socket-utils.c]: Updated PR

**Pull request merged**